thorlcr/thorutil/thbuf.cpp

```diff
@@ -1229,6 +1233,8 @@ class CSharedWriteAheadDisk : public CSharedWriteAheadBase
     Linked<IEngineRowAllocator> allocator;
     Linked<IOutputRowDeserializer> deserializer;
     IOutputMetaData *serializeMeta;
+    size_t tempSizeSpilled = 0;
+    stat_type tempFileElapsedCycles = 0;
```

Comment:
can you change this is a a cycle_t?
logically, it should only be a stat_type once it is converted to nanoseconds and returned in getStatistic